lib/teleterm/clusters/cluster_apps.go

```diff
@@ -170,12 +170,18 @@ func (c *Cluster) reissueAppCert(ctx context.Context, proxyClient *client.ProxyC
 		GCPServiceAccount: "",
 	}
 
-	ws, err := proxyClient.CreateAppSession(ctx, request)
+	rootClient, err := clusterClient.ConnectToRootCluster(ctx)
```

Comment:
I'm wondering how did it work before, `CreateAppSession` could be called on a leaf cluster too?

Reply:
https://github.com/gravitational/teleport/blob/d56ed2c5f88def3d8d0313684c7f867d44d09a65/lib/client/client.go#L773-L795